src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int ARRAY_SIZE = 100;
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
This assignment requires you to split this method into three method calls.
1. Reading data from a file
2. Processing this data and creating a report
3. Write the report to a file.
The end result will be something like:
String data = readFile(fromFileName);
String report = createReport(data);
writeToFile(toFileName, report);

Reply:
But I have only one method in the task, if I remove it and split everything into separate methods the tests will not pass

Reply:
done